shared/src/commonMain/kotlin/com/example/myapplication/shared/welcome/PreviewWelcomeComponent.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.myapplication.shared.welcome
+
+import com.arkivanov.decompose.value.MutableValue
+import com.arkivanov.decompose.value.update
+import com.example.myapplication.shared.getPlatformName
+import com.example.myapplication.shared.welcome.WelcomeComponent.Model
+
+object PreviewWelcomeComponent : WelcomeComponent {
+  override val model = MutableValue(Model())
+
+  override fun onUpdateGreetingText() {
+    model.update { it.copy(greetingText = "Welcome from ${getPlatformName()}") }
```

Comment:
Since JVM previews are not yet interactive, I think we don't need this. We can just put the text in the model. If one needs separate previews with text and without text, then the text can be passed via constructor.

Reply:
Fixed.